data/json/terrain.json

```diff
@@ -8547,6 +8914,82 @@
       ]
     }
   },
+  {
+    "type": "terrain",
+    "id": "t_chimney",
```

Comment:
This should probably be furniture, so that it maintains the roof type underneath it.